src/platform/monitoring/DowntimeNotification/util/helpers.js

```diff
@@ -104,19 +107,36 @@ export function getSoonestDowntime(serviceMap, serviceNames) {
 }
 
 /**
- * Determines whether there is a global downtime in progress
- * @param {object} downtimeWindow Optional arg to set arbitrary downtime
- * @returns {boolean} true if the current time is within the downtime window,
- *     false if not
+ * Retrieves a list of global downtimes from a cached JSON file and gets the
+ * downtime that includes the current time.
+ * @returns {object} A global downtime window that covers the current time
+ *     if it exists and null if not
  */
-export const isGlobalDowntimeInProgress = (
-  downtimeWindow = scheduledDowntimeWindow,
-) => {
-  const { downtimeStart, downtimeEnd } = downtimeWindow;
-
-  return (
-    !environment.isLocalhost() &&
-    moment().isAfter(downtimeStart) &&
-    moment().isBefore(downtimeEnd)
-  );
-};
+export const getCurrentGlobalDowntime = (() => {
+  const BUCKET_BASE_URL = 's3-us-gov-west-1.amazonaws.com';
+
+  const MAINTENANCE_WINDOWS_SUBDOMAINS = Object.freeze({
+    [ENVIRONMENTS.VAGOVDEV]: 'dev-va-gov-maintenance-windows',
+    [ENVIRONMENTS.VAGOVSTAGING]: 'staging-va-gov-maintenance-windows',
+    [ENVIRONMENTS.VAGOVPROD]: 'prod-va-gov-maintenance-windows',
+  });
+
+  const subdomain = MAINTENANCE_WINDOWS_SUBDOMAINS[environment.BUILDTYPE];
+
+  const maintenanceWindowsUrl = subdomain
+    ? `https://${subdomain}.${BUCKET_BASE_URL}/maintenance_windows.json`
+    : null;
+
+  const includesCurrentTime = ({ startTime, endTime }) =>
```

Comment:
Might be worth calling out that this assumes the `maintenance_windows.json` is formatted a specific way.
```json
[
  {
    "startTime": "timestamp",
    "endTime": "timestamp"
  }
]
```